plugins/inputs/stackdriver/README.md

```diff
@@ -0,0 +1,100 @@
+# Stackdriver Input Plugin
+
+Stackdriver scrapes metrics from Google Cloud Monitoring.
+
+### Credentials
+
+This plugin uses a [Google Service Account](https://cloud.google.com/docs/authentication/getting-started) to interacting with the Stackdriver Monitoring API.
+
+### Configuration:
+
+```toml
+[[inputs.stackdriver]]
+  ## GCP Project
+  project = "projects/erudite-bloom-151019"
```

Comment:
Consider documenting that the `projects/` prefix is required by goggle. It took quite some digging for me to find that format used by the stackdriver api is `"projects/{project_id_or_number}"`